src/main/java/kr/hhplus/be/server/order/application/service/OrderEventHandler.java

```diff
@@ -0,0 +1,21 @@
+package kr.hhplus.be.server.order.application.service;
+
+import kr.hhplus.be.server.order.domain.model.OrderCompletedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.event.TransactionPhase;
+import org.springframework.transaction.event.TransactionalEventListener;
+
+@Component
+public class OrderEventHandler {
+
+    @EventListener
+    @TransactionalEventListener(phase = TransactionPhase.AFTER_COMMIT)
```

Comment:
```suggestion
    @TransactionalEventListener(phase = TransactionPhase.AFTER_COMMIT)
```

트랜잭션커밋 이후에 이벤트를 받아서 처리하도록 잘 하셨습니다!
다만 여기에서 @TransactionalEventListener 어노테이션이 @EventListener을 포함하므로 제거해도 될거같습니다.